tools/sigclient/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.18-alpine3.17
```

Comment:
we should update the alpine version here, keep it same as siglens version

Reply:
done. updated it to match siglens repo `FROM golang:1.21-alpine3.18`